tests/schema/tests.py

```diff
@@ -1651,6 +1651,17 @@ def test_order_index(self):
                 index_name = index_name.upper()
             self.assertIndexOrder(Author._meta.db_table, index_name, ['ASC', 'DESC'])
 
+    @unittest.skipUnless(connection.vendor == 'mysql', 'MySQL specific test.')
```

Comment:
`@skipUnlessDBFeature` skips a test if all db features are false. Here we need to skip a test if all db features are true, or execute a test if all db features are false. How `@skipUnlessDBFeature` can be used in a negation scenario?